README.md

```diff
@@ -13,23 +13,21 @@ A Quick Overview of How to Use this Repository
 ----------------------------------------------
 
 To add your project to our build farm, fork this repository, modify the .yaml files, and give us a pull request.
-For Fuerte and older the ROS packages are maintained in the subfolder ``releases`` and ``doc``.
-For Groovy and newer the ROS packages are maintained in the subfolder named after the ROS distribution.
-Mappings for dependencies are maintained in the ``rosdep`` subfolder.
+Mappings for dependencies are maintained in the `rosdep` subfolder of https://github.com/ros/rosdistro
 
-To create jobs on our build farm to build Debian sourcedeb and binarydeb packages add your bloom (or git-buildpackage compatible) repository to the ``ROSDISTRO/release.yaml`` file (or ``releases/ROSDISTRO.yaml`` for Fuerte and older).
+To create jobs on our build farm to build Debian sourcedeb and binarydeb packages add your bloom (or git-buildpackage compatible) repository to the `ROSDISTRO/release.yaml` file.
 Please keep the alphabetic order of the list.
-You can use the script ``scripts/add_release_repo.py`` to perform the insertion.
+You can use the script `scripts/add_release_repo.py` to perform the insertion.
 
-To create jobs on our build farm to perform continuous integration of your repository on every commit add your source repository to the ``ROSDISTRO/source.yaml`` file (or ``releases/ROSDISTRO-devel.yaml`` for Fuerte and older).
+To create jobs on our build farm to perform continuous integration of your repository on every commit add your source repository to the ``ROSDISTRO/source.yaml`` file.
 Please keep the alphabetic order of the list.
-You can use the script ``scripts/add_devel_repo.py`` to perform the insertion.
+You can use the script `scripts/add_devel_repo.py` to perform the insertion.
```

Comment:
I think this script should either be updated or removed as it cannot operate on distribution files (also related to https://github.com/ros/rosdistro/issues/10034)